### PR TITLE
Special handing of Share URL

### PR DESCRIPTION
**Pull request by oo-bldrs**
PR to add parsing and handling of Share URLs and if available in the URL, it will take the camera position and update our view.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -25,14 +22,33 @@ app.post('/rasterize', async (req, res) => {
...
+    const b = parseURLFromBLDRS(url)
+    ifcURL = b.target.url
+
+    if ('c' in b.params) {
+      coordinates = b.params['c'].split(',').map(f => parseFloat(f))
+    }
+  }
+
+  const model = await loadIfcUrl(ifcURL)
   model.position.set(-40, 0, 0)
```
Comment (pablo-mayrgundter) [inline src/server.js]: I think this client may be out of date?  The model position shouldn't be hardcoded

**Review by pablo-mayrgundter — APPROVED** [review]
Looks fine, tho as noted may need a merge.  Please test running the example logo command to make sure it lines up

File: src/server.js (hunk 1 above)
Comment (oo-bldrs) [reply src/server.js]: That's the code that's in the server implementation.

I believe your earlier PR change where you introduced the camera changes only targeted the command line client.

But I'll remove this from the server implementation.

File: src/server.js [hunk 2]
```diff
@@ -25,14 +22,32 @@ app.post('/rasterize', async (req, res) => {
...
+    const b = parseURLFromBLDRS(url)
+    ifcURL = b.target.url
+
+    if ('c' in b.params) {
+      coordinates = b.params['c'].split(',').map(f => parseFloat(f))
+    }
+  }
+
+  const model = await loadIfcUrl(ifcURL)
   scene.add(model)
```
Comment (pablo-mayrgundter) [inline src/server.js:40]: You also need, after model add and before camera pos:

```
// Normalize look and zoom to fit the model in the render frame using
// the same alg as Share.
fitModelToFrame(renderer.domElement, scene, model, camera)
```

Comment (oo-bldrs) [reply src/server.js:40]: Updated.

**Pull request merged**